src/components/pages/Onboarding/Onboarding.tsx

```diff
@@ -277,28 +294,30 @@ class OnboardingView extends React.Component<OnboardingProps, OnboardingState> {
   };
 
   /**
-   * Checks whether the user needs to record a video and returns the CAMERA
-   * OnboardingStep. Otherwise, returns CREATE_ACCOUNT to skip the step.
+   * Checks whether the user has a valid invite token. If not, it sends them to
+   * it requests them for an invite token. If they do, then it checks whether they need
+   * to take an identity video.
    */
-  _validatedCameraStep() {
-    return this.props.hasValidInviteToken &&
-      this.props.inviteVideoIsJoint &&
-      this.state.inviteVideoIsValid
-      ? // The new member does not need to take a verification video
-        // if they have a valid joint invite video.
-        OnboardingStep.CREATE_ACCOUNT
-      : OnboardingStep.CAMERA;
+  _validatedInviteTokenStep() {
+    return this.props.hasValidInviteToken && this.state.inviteVideoIsValid
+      ? // The new member is not requested to input an invite token
+        // if they already have a valid one.
+        // Note, Previously we would send the user directly to CREATE_ACCOUNT
+        // if the onboarding video was joint. Now we require an identity video from
```

Comment:
I do like the idea of everyone having an identity video, but I also don't want to add it as a mandatory extra step for someone who already made a video with their friend as that feels very heavy for this early stage ... would be nice to let people see what Raha is about ASAP and then allow and encourage members to update their identity video at any time.

Reply:
let's give it a shot and can easily roll back if it turns out to be a sticking point